Formula/p/pdfpc.rb

```diff
@@ -18,34 +18,33 @@ class Pdfpc < Formula
     sha256 x86_64_linux:   "aa2f690eacb00966ce31538971a580c18fabbefd1c222ddc69918c83a396e561"
   end
 
-  head do
-    url "https://github.com/pdfpc/pdfpc.git", branch: "master"
-
-    depends_on "discount"
-    depends_on "json-glib"
-    depends_on "libsoup@2"
-    depends_on "qrencode"
-
-    on_linux do
-      depends_on "webkitgtk"
-    end
-  end
-
   depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
   depends_on "vala" => :build
+  depends_on "discount"
   depends_on "gstreamer"
   depends_on "gtk+3"
+  depends_on "json-glib"
   depends_on "libgee"
   depends_on "librsvg"
+  depends_on "libsoup"
   depends_on "poppler"
+  depends_on "qrencode"
+
+  on_linux do
+    depends_on "webkitgtk"
```

Comment:
```
-- Checking for module 'webkit2gtk-4.0'
  --   No package 'webkit2gtk-4.0' found
  CMake Error at /home/linuxbrew/.linuxbrew/Cellar/cmake/3.27.8/share/cmake/Modules/FindPkgConfig.cmake:607 (message):
    A required package was not found
  Call Stack (most recent call first):
    /home/linuxbrew/.linuxbrew/Cellar/cmake/3.27.8/share/cmake/Modules/FindPkgConfig.cmake:[82](https://github.com/Homebrew/homebrew-core/actions/runs/6986179004/job/19011401575?pr=155471#step:4:83)9 (_pkg_check_modules_internal)
    src/CMakeLists.txt:33 (pkg_check_modules)
```

Reply:
Interestingly, we install `include/webkitgtk-4.1/` / `libexec/webkit2gtk-4.1/`, not version `4.0` ..

`pdfpc` is expliciltely looking for `4.0` here: https://github.com/pdfpc/pdfpc/blob/14344bc203cfff71a1b72deb161e003c9b480bb2/src/CMakeLists.txt#L35-L38

Reply:
Good find, let me raise it up.